backend/app/interactem/app/api/routes/operators.py

```diff
@@ -1,10 +1,7 @@
-import uuid
-from typing import Any
 
-from fastapi import APIRouter, HTTPException
-
```

Comment:
can you run `npx biome lint --fix && px biome format --write .`, too? from `frontend/interactem` dir.

Reply:
Sure

Reply:
done